src/main/resources/lang/en_US.json

```diff
@@ -449,6 +449,11 @@
         "disabled": "Magma boss timer messages will no longer be logged.",
         "enabled": "Magma boss timer messages will now be logged."
       },
+      "version": {
+        "help": "Prints the mod's version",
+        "hoverText": "Click to put in chat so you can copy.",
```

Comment:
Can I see a screenshot of this? It may look weird depending on how it's displayed and the text that appears below